### PR TITLE
terminal config

### PR DESCRIPTION
**Pull request by frederik-uni**
Release Notes:

- terminal header button config(show buttons while unfocused, disable buttons, change add button behavior, change order of buttons)

### CODE REVIEW

File: assets/settings/default.json [hunk 1]
```diff
@@ -688,6 +688,18 @@
...
+    // Whether to show the terminal maximize button above the terminal
+    "maximize_button": false,
+    // Whether to show a drop down for spawning tasks & creating new terminals or creating a new terminal
+    "spawn_task_option_button": true,
+    // Whether to show the inline assistent button above the terminal
+    "inline_assist_button": true,
+    // Whether to show the buttons above the terminal only when terminal in focus
+    "require_focus_for_buttons": true,
+    // In which order the buttons above the terminal are
+    "arrange_buttons": ["InlineAssist", "New", "Minimize", "Maximize"],
```
Comment (maxdeviant) [inline assets/settings/default.json:702]: I don't think we should add a setting to arrange the order of the buttons.

We don't really do anything like this elsewhere, and it seems like it's going to restrict what we're able to do with the UI.

Comment (frederik-uni) [reply assets/settings/default.json]: it does, it changes the order of the button but the functionality stays the same but ok

**Comment by maxdeviant** [comment]
Thank you for the PR, but we're not going to move forward with these settings additions at this time.

**Comment by frederik-uni** [comment]
@maxdeviant will the buttons that are toggable right now at least stay or will this feature be removed in the future?

**Comment by maxdeviant** [comment]
> @maxdeviant will the buttons that are toggable right now at least stay or will this feature be removed in the future?

No plans to change those buttons, as far as I'm aware.

**Pull request closed without merging**